lib/utils/options.dart

```diff
@@ -403,6 +404,10 @@ class HtmlToolbarOptions {
   final Color? buttonSelectedBorderColor;
   final BorderRadius? buttonBorderRadius;
   final double? buttonBorderWidth;
+
+  /// Allow the user to choose an image from their device when
```

Comment:
Can you note here that this does not disable inserting images via url?

Reply:
Done !